### DISCUSSION TITLE
I can't use tailwind postcss in vs code

### DISCUSSION THREAD

**Discussion by MinThantKyaw7** [tailwindlabs/tailwindcss#11329] [Help]
I can't use tailwind css in vs code
It tell "missing script"dev" "
when I run "npm run dev"
What's wrong?

**Comment by wongjn** [comment]
It seems like you are missing the `dev` script in your project's `package.json`, see [npm documentation on `scripts`](https://docs.npmjs.com/cli/v9/using-npm/scripts) for further details.

**Reply by MinThantKyaw7** [reply]
It can't work but it tells postcss.config.js has unknown variable

**Reply by wongjn** [reply]
I'm sorry, I don't quite understand what you are trying to convey. Could you elaborate more on what you are attempting to achieve, what you expect to happen and what happens in reality please?

**Reply by MinThantKyaw7** [reply]
![IMG20230531204735.jpg](https://github.com/tailwindlabs/tailwindcss/assets/76880381/af7c7b8e-8f01-4cfc-87de-b0bdb0ee08df)

**Reply by MinThantKyaw7** [reply]
![IMG20230531204742.jpg](https://github.com/tailwindlabs/tailwindcss/assets/76880381/197f5725-2151-48a7-9bc1-1d4b0fe4e512)

**Reply by wongjn** [reply]
Thank you for providing those screenshots. My original reply still applies: https://github.com/tailwindlabs/tailwindcss/discussions/11329#discussioncomment-6050214

**Comment by f3bw** [comment]
Regarding the other discussion in this thread. Adding a line with something like: `"dev": "next dev",` in the _scripts_ section in the _package.json_ file might help. _(I am using next in my set up)_

Another solution is to install tailwind and all other packages with Yarn. 
(I am a strong Yarn enthousiast)

So instead of using `npm run dev` it would be `yarn dev`

For installation use: https://tailwindcss.com/docs/installation and replace all the `npm` for `yarn`
